roles/tas_single_node/templates/manifests/rekor/backfill_redis.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+LAST_INDEX_FILE="/tmp/rekor_last_index"
+
+endIndex=$(curl -sS http://localhost:3001/api/v1/log | jq -r '.treeSize - 1')
+if [ "$endIndex" -lt 0 ]; then
+  echo "info: no rekor entries found"
+  exit 0
+fi
+
+if [ -f "$LAST_INDEX_FILE" ]; then
+  startIndex=$(cat "$LAST_INDEX_FILE")
+else
+  startIndex=0
+fi
+
+if [ "$startIndex" -gt "$endIndex" ]; then
+  startIndex=0
+fi
+
+if [ "$startIndex" -le "$endIndex" ]; then
+  podman run --rm --name backfill_redis \
+    -v "{{ tas_single_node_certs_dir }}/ingress-rekor.pem:/usr/local/share/ca-certificates/ingress-rekor.pem:ro" \
+    -e SSL_CERT_FILE=/usr/local/share/ca-certificates/ingress-rekor.pem \
```

Comment:
Is this the correct filepath? iirc the path for ingress certs is `/etc/rhtas/certs/ingress-{{component}}.pem`

Reply:
Would also prefer this to be a var instead of hardcoded if above comment is wrong ^

Reply:
The file paths here are correct, {{ tas_single_node_certs_dir }}/ingress-rekor.pem refers to the file path in the rhel machine which is mounted inside the bacfill_redis pod to the /usr/local/share/ca-certificates/ingress-rekor.pem file using the -v (volume) flag 

eg     -v "(file to mount):(path to mount it to)" \

I think that variables here dont really make sense as /usr/local/share/ca-certificates is the default for most machines when it comes to ca certificates